Find a functionally equivalent code:com.android.internal.content.NativeLibraryHelper.removeNativeBinariesLI:COMMENT
Method Modifier: public      static      internal    
Comment:// Convenience method to call removeNativeBinariesFromDirLI(File)

@@ -1,3 +1,5 @@
 {
-    return removeNativeBinariesFromDirLI(new File(nativeLibraryPath));
+    if (nativeLibraryPath == null)
+        return;
+    removeNativeBinariesFromDirLI(new File(nativeLibraryPath), false);
 }

